Utils/Dataflow/019_esFormat/README

```diff
@@ -40,3 +40,15 @@ For input it is OK, but for output it would be better to take <index info> and
   be critical for now, but something tells me that it will.
 This task requires accurate EOMessage and EOProcess markers handling (see the
   `kafka-dataflow-markers` branch).
+
+Samples
+-------
+The 'output' directory contains two pairs of samples:
+
+2016 data:
+tasks.ndjson (from 016's 2016 sample, 025 was skipped)
```

Comment:
Er, 25 was skipped before 16, right? So no need to mention it here... I mean -- according to the `/Utils/Dataflow/README`, it should come from 16 -- and it does; being here you must not worry about "how 016'th sample was created". Even if it was written by scratch manually, it is not your problem, if you are here: the one who wrote it is responsible for making it a correct sample.

Reply:
Fair enough, but will be obsolete per comment in #201.

Reply:
Done.